README.md

```diff
@@ -1,10 +1,14 @@
 # judoscale-python
 
-This is the official Python adapter for [Judoscale](https://elements.heroku.com/addons/judoscale). You can use Judoscale without it, but this gives you request queue time metrics and job queue time (for supported job processors).
+This is the official Python adapter for
+[Judoscale](https://elements.heroku.com/addons/judoscale). You can use Judoscale
+without it, but this gives you request queue time metrics and job queue time
+(for supported job processors).
 
 ## Installation
 
-Add judoscale-python to your <code>requirements.txt</code> file:
+Add judoscale-python to your <code>requirements.txt</code> file or the
+equivalent:
```

Comment:
It's kind of a bummer that the README must adhere to the 88-char line length, especially since Black isn't autoformatting it (at least for me). Did you have to make these changes manually, or does it automatically format for you? Note that Black *is* autoformatting Python files for me, just not the README.

Same question about comments within Python files. I noticed you broke them to adhere to the line length, but it doesn't appear to be automated.

I did a quick search to have Flake8 ignore comments, but I couldn't find anything. It's not something worth much time at all, but I wanted to bring it up in case you have an easy solution.

Reply:
Oh.. the README doesn't have to have 88-char lines — that was me formatting it manually. I think I did it out of habit. Will revert this.

Reply:
Re: Flake8, Black and comments — yes, Black won't reformat comments and Flake8 will flag long lines. You could append `#  noqa: E501` which will cause Flake8 to ignore long lines. But I have always just reformatted long comments manually with `Vgq` (Vim keybindings).

Reply:
I love that I'm still learning new Vim tricks after 15 years. 😂

No need to ignore long lines. It's a minor thing that I can live with.